apps/confidential/confidential_test.go

```diff
@@ -66,3 +75,93 @@ lQFhpxvPgPNBuxVRnsxoH/sLOA==
 		t.Fatalf("TestCertFromPEM: got nil key, want key != nil")
 	}
 }
+
+func fakeClient() (Client, error) {
+	cred, err := NewCredFromSecret("fake_secret")
+	if err != nil {
+		return Client{}, err
+	}
+	client, err := New("fake_client_id", cred, WithAuthority("https://fake_authority/fake"))
+	if err != nil {
+		return Client{}, err
+	}
+	client.base.Token.AccessTokens = &fake.AccessTokens{
```

Comment:
Note that I faked the min set of fields in all these responses to get things working.

Reply:
This should be better.